src/tag/tag.style.ts

```diff
@@ -33,91 +31,97 @@ export const Wrapper = styled.div<StyleProps>`
+                    borderColor = Colour["border-success"];
+                    color = Colour["text-success"];
+                    hoverBackgroundColor = Colour["bg-success-hover"];
                     break;
                 case "yellow":
-                    backgroundColor = V2_Color.Validation.Orange.Background;
-                    borderColor = V2_Color.Validation.Orange.Border;
-                    color = V2_Color.Validation.Orange.Text;
+                    backgroundColor = Colour["bg-warning"];
+                    borderColor = Colour["border-warning"];
+                    color = Colour["text-warning"];
+                    hoverBackgroundColor = Colour["bg-warning-hover"];
                     break;
                 case "red":
-                    backgroundColor = V2_Color.Validation.Red.Background;
-                    borderColor = V2_Color.Validation.Red.Border;
-                    color = V2_Color.Validation.Red.Text;
+                    backgroundColor = Colour["bg-error"];
+                    borderColor = Colour["border-error"];
+                    color = Colour["text-error"];
+                    hoverBackgroundColor = Colour["bg-error-hover"];
                     break;
                 case "blue":
-                    backgroundColor = V2_Color.Validation.Blue.Background;
-                    borderColor = V2_Color.Validation.Blue.Border;
-                    color = V2_Color.Validation.Blue.Text;
+                    backgroundColor = Colour["bg-info"];
+                    borderColor = Colour["border-info"];
+                    color = Colour["text-info"];
+                    hoverBackgroundColor = Colour["bg-info-hover"];
                     break;
                 default:
-                    backgroundColor = V2_Color.Neutral[8];
-                    borderColor = V2_Color.Neutral[5];
-                    color = V2_Color.Neutral[1];
-                    boxShadow = `0px 0px 4px 1px rgba(164, 164, 164, 0.50)`;
+                    backgroundColor = Colour.bg;
+                    borderColor = Colour.border;
+                    color = Colour.text;
+                    hoverBackgroundColor = Colour["bg-hover-neutral"];
                     break;
             }
         }
         return css`
             background: ${backgroundColor};
-            border: 1px solid ${borderColor};
+            border: ${Border["width-010"]} ${Border.solid} ${borderColor};
             color: ${color};
-
-            @media (hover: hover) {
-                &:hover {
-                    box-shadow: ${props.$interactive ? boxShadow : "none"};
-                }
+            &:hover {
+                ${props.$interactive && hoverBackgroundColor
+                    ? `background: ${hoverBackgroundColor(props)};`
+                    : ""}
+                ${props.$interactive && hoverBorderColor
+                    ? `border-color: ${hoverBorderColor(props)};`
+                    : ""}
```

Comment:
instead of ternary with empty string, does it work to do something like `interactive && hoverBorderColor && `border-color: ${hoverBorderColor(props)}`?